### ISSUE TITLE
Wrong DDL generated in Oracle when translating integer types

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#10566]
Translating this to Oracle:

```sql
CREATE TABLE t (
  a TINYINT,
  b SMALLINT,
  c INT,
  d BIGINT
);
```

Yields:

```sql
create table T (
  A number null,
  B number null,
  C number null,
  D number null
)
```

When it should yield:

```sql
create table T (
  A number(3) null,
  B number(5) null,
  C number(10) null,
  D number(19) null
)
```

**Comment by lukaseder** [comment]
This could be seen as an incompatible change if someone relies on the current behaviour, which is why I won't backport this. Fixed for jOOQ 3.14.0

**Comment by lukaseder** [comment]
Will look into https://github.com/jOOQ/jOOQ/issues/10376 again, for a more object-oriented solution to these kinds of issues
